app/Jobs/ImportWikiDocumentation.php

```
@@ -178,11 +187,29 @@ protected function toTitleCase(string $text): string
      */
     protected function updateInternalLinks(string $content): string
     {
-        // Get the site URL
         $siteUrl = rtrim(config('app.url'), '/');
 
-        // Pattern to match wiki links (both relative and absolute)
-        // This will match links like: [text](page-slug) or [text](parent/page-slug)
+        // Extract the wiki base URL from the package wiki_url for absolute link matching
+        // e.g., https://github.com/owner/repo/wiki -> matches https://github.com/owner/repo/wiki/Page-Name
+        $wikiBase = rtrim($this->package->wiki_url, '/');
+        $escapedWikiBase = preg_quote($wikiBase, '/');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$escapedWikiBase'. (undefined)

(UnusedLocalVariable)